udapi/block/ud/la/addmwt.py

```diff
@@ -0,0 +1,58 @@
+""" Block ud.la.AddMwt for heuristic detection of multi-word (PRON + cum, nonne) and abbreviations-dots tokens. """
+import udapi.block.ud.addmwt
+
+MWTS = {
+    'mecum':   {'lemma': 'ego cum', 'form': 'me cum', 'feats': 'Case=Abl|Gender=Masc|Number=Sing AdpType=Post|Clitic=Yes'},
+    'tecum':  {'lemma': 'tu cum', 'form': 'te cum', 'feats': 'Case=Abl|Gender=Masc|Number=Sing AdpType=Post|Clitic=Yes'},
+    'nobiscum':   {'lemma': 'nos cum', 'form': 'nobis cum', 'feats': 'Case=Abl|Gender=Neut|Number=Plur AdpType=Post|Clitic=Yes'},
+    'vobiscum': {'lemma': 'vos cum', 'form': 'vobis cum', 'feats': 'Case=Abl|Gender=Masc|Number=Plur AdpType=Post|Clitic=Yes'},
+    'uobiscum': {'lemma': 'uos cum', 'form': 'uobis cum', 'feats': 'Case=Abl|Gender=Masc|Number=Plur AdpType=Post|Clitic=Yes'},
+    'secum': {'lemma': 'sui cum', 'form': 'se cum', 'feats': 'Case=Abl|Gender=Masc AdpType=Post|Clitic=Yes'}, # can be singular or plural
+}
+
+# shared values for all entries in MWTS
+for v in MWTS.values():
+    v['upos'] = 'PRON ADP'
+    # v['xpos'] = '' # treebank-specific
+    v['deprel'] = 'obl case'
+    v['main'] = 0
+    v['shape'] = 'subtree'
+	   
+
+class AddMwt(udapi.block.ud.addmwt.AddMwt):
+    """Detect and mark MWTs (split them into words and add the words to the tree)."""
+
+    def multiword_analysis(self, node):
+        """Return a dict with MWT info or None if `node` does not represent a multiword token."""
+        analysis = MWTS.get(node.form.lower(), None)
+        if analysis is not None:
+            return analysis
+            
+        if node.form.endswith('.') and len(node.form) > 1 and node.form != '...':
+            dic = {
+                'form': 'x .',
+                'lemma': '* .',
+                'upos': '* PUNCT',
+                'xpos': '_ _',
+                'feats': '* _',
+                'deprel': '* punct',
+                'main': 0,
+                'shape': 'subtree'
+            }
+            forma = node.form[:-1] + ' .'
+            dic.update(form = forma)
+            return dic
+        elif node.lemma == 'nonne':
```

Comment:
Alternatively, this could be added to `MWTS` and the code below could be adapted to
```
for v in MWTS.values():
    if 'upos' not in v:
        v['upos'] = 'PRON ADP'
    ...
```
which could be useful (more compact&readable code) if there are more such non-PRON-ADP MWTs. If it is just this one, both versions seem OK.

There is a small difference that this code checks `node.lemma=="nonne"`, but the general code would check `node.form.lower()=="nonne"`, but I guess it is equivalent given the lemmatization used in the data.